Tests/UnitTests/Core/Surfaces/LineSurfaceTests.cpp

```diff
@@ -104,7 +111,8 @@ BOOST_AUTO_TEST_CASE(LineSurface_allNamedMethods_test) {
   // globalToLocal()
   Vector3 gpos{0., 1., 0.};
   const Vector3 mom{20., 0., 0.};  // needs more realistic parameters
-  Vector2 localPosition = line.globalToLocal(tgContext, gpos, mom).value();
+  Vector2 localPosition =
+      line.globalToLocal(tgContext, gpos, mom.normalized()).value();
```

Comment:
I guess the momentum normalization should happen in `globalToLocal` to avoid the use of the function with unnormalized momenta

Reply:
not sure. I recently changed this so the surface base class explicitly says that this is a unit direction. my intention was to make the interface clear:
- we do not depend on the magnitude
- we need to have a direction
- we will no recalculate the length of this vector

Reply:
Ah, ok, now I see that the normalization does not matter. Why do you normalize in the unit test then?

Reply:
sometimes it does not matter, sometimes it does. the contract of the base class method requires a normal vector so I would always provide a normal vector